ecommerce/orders/models.py

```diff
@@ -0,0 +1,84 @@
+from django.db import models
+
+from basket.models import Basket
+from core.models import BaseAbstractModel
+from django.utils.translation import gettext_lazy as _
+
+# Create your models here.
+from customers.models import City, Customer
+
+
+class BillingAddress(BaseAbstractModel):
```

Comment:
Could create AbstractAddress class and extend Billing and Shipping class from it.